static/js/pages/instances/InstanceDetail.tsx

```diff
@@ -31,11 +60,27 @@ const InstanceDetail: FC = () => {
 
   return (
     <BaseLayout
-      title={`Instance details for ${name}`}
+      title={
+        <List
+          inline
+          items={[
+            <span key="name">{name}</span>,
+            <span key="status" className="u-instance-status">
+              {instance.status}
+            </span>,
+            <StartStopInstanceBtn
```

Comment:
I think a dense button might be better fitting the text height.

Reply:
You mean in `InstanceDetailPanel` or here? I think here is fine - the other controls are also not dense.

Reply:
I meant the detail page, but ymmw - Just thought it should be part of the Title and match its height. But consistency with the other buttons on the right is also a valid aspect.